3rd-party/beaverbuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Beaverbuilder Compatibility.
+ */
+class BeaverBuilderCompat {
+
+	function __construct() {
+		add_action( 'init', array( $this, 'masterbar_refresh' ) );
+	}
+
+	/**
+	 * If masterbar module is active force BeaverBuilder to refresh when publishing a layout.
+	 */
+	function masterbar_refresh() {
```

Comment:
Do you think you could name this a bit differently, since we are refreshing the builder, and not the masterbar?

Reply:
Not a problem.